### PR TITLE
chore: update DASH license Plugin config + fix typo

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description

- modify dash licence plugin information to current released plugin version 
- fix typos

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/release/trg-7/trg-7-04.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ Make sure to also include test dependencies. For a maven-based java project you
       <plugin>
```
Comment (SebastianBezold) [inline docs/release/trg-7/trg-7-04.md:73]: I think this is misleading/wrong. The plugin section is included in `<build></build>`. Here it seems like the plugin section is on the same level as `<pluginRepositories>`. I think this should be changed with this PR as well

**Review by SebastianBezold — APPROVED** [review]
I think this is a good change. The mentioned improvement can also be done in a separate PR

File: docs/release/trg-7/trg-7-04.md (hunk 1 above)
Comment (FaGru3n) [reply docs/release/trg-7/trg-7-04.md:73]: like this?
image

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, well seen!

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**